pyuvsim/analyticbeam.py

```diff
@@ -29,23 +37,23 @@ def peak_normalize(self):
     @profile
     def interp(self, az_array, za_array, freq_array):
         """
-        Interpolate beam to given az, za locations (in radians).
+        Evaluate the primary beam at given az, za locations (in radians).
+        (similar to UVBeam.interp)
 
-        Mirrors the interp function of UVBeam objects.
         Args:
-            az_array: az values to interpolate to in radians (same length as za_array)
-                The azimuth here has the UVBeam convention: North of East(East=0, North=pi/2)
-            za_array: za values to interpolate to in radians (same length as az_array)
-            freq_array: frequency values to interpolate to
+            az_array: az values to evaluate (same length as za_array)
+            za_array: za values to evaluate (same length as az_array)
+            freq_array: frequency values
```

Comment:
I'd like to keep at least the args entries the way they were before.

Reply:
I restored the args text as before, except I replaced "interpolate" with "evaluate." A minor nitpick, but since this is an analytic beam we're not interpolating but actually returning an exact function value. Is that okay?

Reply:
Yes, thanks! I just didn’t want to lose the units and convention notes